installation/resources/crds/telemetry/logpipelines.crd.yaml

```diff
@@ -137,7 +137,7 @@ spec:
                     description: Configures an output to the Kyma-internal Loki instance.
                       [Fluent Bit grafana-loki output](https://grafana.com/docs/loki/v2.2.x/clients/fluentbit/).
                       **Note:** This output is considered legacy and is only provided
-                      for backward compatibility with the [deprecated](https://kyma-project.io/blog/2022/11/2/loki-deprecation/)
+                      for backward compatibility with the [deprecated](https://github.com/kyma-project/website/blob/main/content/blog-posts/2023-01-17-release-notes-2.10/index.md#deprecation-of-monitoring)
```

Comment:
I haven't dug deep into the topic so maybe I'm wrong, but is this the right link? Shouldn't it be: https://github.com/kyma-project/website/blob/main/content/blog-posts/2022-11-02-loki-deprecation/index.md?

Reply:
absolutely! I must have mis-clicked. Thanks for spotting this.